### PR TITLE
Suggestions from @ekr

### LINKED ISSUE #26: Email suggestions from @ekr
[Email suggestions](https://mailarchive.ietf.org/arch/msg/gendispatch/x1JcEWRYy8_NXE07DqdkCWXr1lQ) from @ekr:
```
I think this is a good idea and this draft is a good start. I have
made a few comments below, but I want to make one general suggestion
for enhancement.

In my experience one of the things that makes moderation difficult
is when each moderation decision is itself subject to extensive
debate in the same venue as it is happening. This constitutes a
DoS on the venue as well as discouraging strong moderation. I
think it would be good to establish a separate venue for debating
moderation discussions (e.g., an issue tracker or something)
that is distinct from the lists being moderated. This will allow
for public debate where necessary but not subject everyone else to it.


S 4.1.
   The moderator team SHALL operate according to a consistent and
   uniform set of criteria, processes, and actions.  The moderator team
   SHALL independently define and execute their role.  They SHALL
   maintain a public set of moderation criteria, processes, actions, and
   other material that allows the community to understand and comment on
   the role of the team.  The moderator team SHOULD consider adopting
   moderation criteria, processes, and actions that other technical
   communities have found suitable.  The moderator team's criteria and
   processes SHALL be developed with community input, but they are not
   expected to be documented in the RFC series.

So these do not require approval of IAB or IESG?

   The moderator team MAY initiate moderation actions by itself;
   individual participants SHOULD also alert the team to disruptive
   behavior they observe.  Participants should be able to contact the
   moderator team in ways that are, ideally, integrated into the various
   participation channels the IETF offers.

I think you should clarify that this is confidential.


Maybe I'm missing it, but it's not clear to me what the *powers*
of moderators are. For instance, can they ban someoen entirely
from a list? From an IETF list? It just seems to say "moderation
actions". FWIW my view is they should have quite broad powers.



S 4.2.

   Because the IESG and IAB are in the appeals chain for moderator team
   decisions (see Section 4.3), the IETF Chair SHOULD NOT appoint a
   moderator who is serving on the IESG or IAB.

I think this should probably be a MUST and I would change it to
forbid someone becoming a moderator and then joining the I*.

   Because the IESG and IAB are in the appeals chain for moderator team
   decisions (see Section 4.3), serving members of the IESG and IAB
   MUST NOT serve as moderators.

I don't see a clause that allows replacing moderators. That seems
like it needs to exist.

S 4.4.
   Due to the global nature of the IETF, the membership of this team
   SHOULD reflect a diversity of time zones and other participant
   characteristics that lets it operate effectively around the clock and
   throughout the year.  Team diversity is also important to ensure any
   participant observing problematic behavior can identify a moderator
   they feel comfortable contacting.

This seems to implicitly assume a very high level of responsiveness
(24x7). I'm not sure that's necessary but if you think it is, then
you should say so.
```

### PR DESCRIPTION
**Pull request by larseggert**
Fixes #26

### CODE REVIEW

**Comment by russhousley** [comment]
I worry about the use of the word "uniform".  We need to recognize that the energy that a moderator team member has will vary.  $dayjob will place different demands on different days.  I'd prefer wording that aspires to the ideal, but recognizes reality too.

**Comment by becarpenter** [comment]
I don't see Russ's problem with 'uniform' because it refers to criteria etc., not to the moderators themselves. It's true that availability and response times will vary, but that seems like a separate issue.

**Review by becarpenter — DISMISSED** [review]
wfm

**Comment by larseggert** [comment]
Regarding "uniform", I tend to agree with Brian. The criteria should be uniform across the org and its channels, but there needs to be an acceptance that moderation actions based on the criteria may have some variance, given that people are involved.

File: draft-ecahc-moderation.md [hunk 1]
```diff
@@ -227,15 +217,40 @@ their own judgment and community suggestions.
 > or whether chair and list admins should only inform the moderator
 > team of potentially disruptive behavior and let them deal with it.
 
+## Transparency
+
+The moderator team SHALL operate according to a consistent and uniform
```
[the remark this answers is not in the document]
Comment (becarpenter) [reply draft-ecahc-moderation.md:222]: wfm

**Pull request merged**